articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -85,6 +81,7 @@ Once the resources are created, add the following environment variables. These w
 Open *Program.cs* and update the file with the following code.
 
 ```csharp
```

Comment:
Can you please clean up the usings? Since we don't parse the push notification anymore, I suppose some usings are not needed.